stc_game/Assets/Scripts/SceneControl/PersistentScene/PersistentScene.cs

```diff
@@ -37,54 +80,23 @@ private void Start()
+        
+        //actionBar = FindObjectOfType<ActionBarController>();  => This returns null every time, not sure why
 
         dialogueUI = GameObject.FindObjectOfType<DialogueUI>();
         DialogueManager.instance.dialogueUI = PersistentScene.Instance.dialogueUI;
 
         userService.LoadUserCallback += HandleLoadUserCallback;
-        userService.GetUser(AuthService.Instance.authUser.sub);
 
-        // Substitue GameCharacter to be replaced by GameCharacter data from database
-        // This is just for testing
-        GameCharacter = new GameCharacter(
-                                            "WarriorTest",
-                                            HeroClass.Warrior,
-                                            new Stats()
-                                            {
-                                                level = 1,
-                                                XP = 0,
-                                                gold = 0,
-                                                maxHP = 500,
-                                                maxAP = 150,
-                                                currentHP = 500,
-                                                currentAP = 150,
-                                                strength = 10,
-                                                intellect = 5,
-                                                dexterity = 7,
-                                                attack = 15,
-                                                meleeCritPower = 2,
-                                                defense = 9,
-                                                abilityAttack = 10,
-                                                abilityCritRate = 0.25,
-                                                abilityCritPower = 2,
-                                                meleeCritRate = 0.25,
-                                                dodgeRate = 0.15,
-                                                movementSpeed = 7,
-                                                dead = false,
-                                                nextLevelXP = 100
-                                            }
-                                            ,
-                                            new List<string>()
-                                            {
-                                               "RegenerateAP",
-                                               "IronSkin",
-                                               "Savagry",
-                                               "Cleave",
-                                               "Sprint"
-                                            }
-                                            );
+        if (UserService.Instance.user._id != "")
+        {
+            userService.GetUser(AuthService.Instance.authUser.sub);
+        }
+        else // for now so we can start from persistent scene without having to login

```

Comment:
yeah this is my bad, I need to get a local storage for the user or a mock user data that can be loaded here before the scene

Reply:
np, this was just a quick temporary fix